dts/bindings/display/sitronix,st7735r.yaml

```diff
@@ -115,3 +115,7 @@ properties:
       type: int
       default: 0x07
       description: Display Inversion Control
+
+    invert-color:
+      type: boolean
+      description: Display color inversion
```

Comment:
Since you are touching it, can you please fix description from `invctr` above to "Display Inversion mode"? If so please do it in separate commit.

The name and description of property you would like to add should probably be:

```
    inversion-on:
      type: boolean
      description: Enable Display Inversion
```

Reply:
Splitting commit and Fix it.
Changing variable name as well.

Reply:
>Since you are touching it, can you please fix description from `invctr` above to "Display Inversion mode"? If so please do it in separate commit.

You do not need to split your commit. I asked if you could fix the description of existing `invctr` property because it is a bit confusing now, if so, it should be done in separate commit.

Reply:
I got it.
I updated description of 'invctr' with separeted commit with  adding short summary of datasheet's description.